nncf/hw_configs/cpu.json

```diff
@@ -26,12 +26,12 @@
                 "mode": "symmetric",
                 "level_low": -128,
                 "level_high": 127,
-                "granularity": "perchannel"
+                "granularity": ["perchannel", "pertensor"]
```

Comment:
How about other configs? Since it is obvious that pertensor is automatically supported if perchannel is in the support and we would like just formally identify this in the confugs, it is worth changing all the impacted configs.

Reply:
Done